checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitVisitor.java

```diff
@@ -349,6 +349,13 @@ public Void visitNewArray(NewArrayTree tree, Void p) {
                     type.toString());
         }
 
+        List<? extends AnnotationMirror> annotations =
+                TreeUtils.annotationsFromArrayCreation(tree, 0);
```

Comment:
Can you add a comment why this doesn't use `type`? That is already the type of the array creation. Was it because `@PolyNull` would already be substituted?